website/publish-gh-pages.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const branch = process.env.TRAVIS_BRANCH;
+const isPullRequest = process.env.TRAVIS_PULL_REQUEST;
+const isTravis = process.env.TRAVIS;
+const remoteBranch = 'git://github.com/facebook/react-native.git';
+require('shelljs/global');
+
+if (!which('git')) {
+  echo('Sorry, this script requires git');
+  exit(1);
+}
+
+let version;
+if (branch.indexOf('-stable') !== -1) {
+  version = branch.slice(0, branch.indexOf('-stable'));
+}
+
+if (exec(`node ./server/generate.js`).code !== 0) {
+  echo('Error: Generating HTML failed');
+}
+
+console.log(`isPullRequest ${isPullRequest} isTravis ${isTravis } version ${version}`);
```

Comment:
Not sure if we node supports it yet but I really like

console.log({ isPullRequest, isTravis, version })

For quick logging